src/inverseRobot.js

```diff
@@ -24,7 +24,30 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  let robotInversed = {};
+
+  // Checking for repeated values inside a source object
+  for (const key2 in robot) {
+    for (const key3 in robot) {
```

Comment:
you don't need to use nested loops in this task

Reply:
simplify this part, please

Reply:
Thank you for your suggestions. I decided to use .values and .sort methods instead of nested loops.